spynl/tests/test_endpoint_availability.py

```diff
@@ -39,6 +22,7 @@ def non_production_endpoint(ctx, request):
 
     monkeypatch.setattr(endpoints, 'main', patched_main)
 
+    settings['spynl.spynl_environment'] = 'production'
```

Comment:
This is out of scope of this, but IMO if we set the environment to production in non-production settings, we should seriously consider to banish any if statements asking for why env we are in.